sql/trust_list.sql

```diff
@@ -0,0 +1,68 @@
+CREATE DATABASE  IF NOT EXISTS `dspdb` /*!40100 DEFAULT CHARACTER SET utf8 */;
```

Comment:
What's going on with this table~? Is this required too?

(Also, trusts go higher than level 99, and don't have elements. If 'time' is meant to be recast time, these are all currently set to 0 here.)

Reply:
he has multiple new tables that don't need to exist, as we have existing tables the data can go in after they start using the fields in mob_pools: skill lists and spell lists can be same as existing mobs and pets

Reply:
this is why I wasn't going to full on review any pr that doesn't start using pool info.